roboteam_ai/src/skills/PenaltyKeeper.h

```diff
@@ -24,6 +24,9 @@ class PenaltyKeeper : public Skill  {
         control::BasicPosControl gtp;
         PenaltyState updateState(PenaltyState currentState);
         bool isBallShot();
+        //testing:
```

Comment:
meh. remove these.

Reply:
I will 100% need these when we have real robots. Don't see the point of removing them to add them again.

Reply:
removed them but this results in a lot of commented code, pick your poison.